l10n_ch_dta/README.rst

```diff
@@ -0,0 +1,59 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==========================
+Switzerland DTA
+==========================
+
+This module implements DTA. it is recommended to use with l10nch_payment_register module
```

Comment:
In fate l10n_ch_payment_register is a dependency, you might want to rephrase this.